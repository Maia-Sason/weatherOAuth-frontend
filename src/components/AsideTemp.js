const AsideTemp = () => {
  return <> </>;
};

export default AsideTemp;
