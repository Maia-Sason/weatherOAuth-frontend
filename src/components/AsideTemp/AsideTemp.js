import { Typography, makeStyles, Box } from "@material-ui/core";

const useStyles = makeStyles(() => ({
  text: {
    width: "fit-content",
    fontSize: "2em",
    fontWeight: "100",
    margin: "5px",
  },
}));

const AsideTemp = ({ text, temp }) => {
  const classes = useStyles();
  return (
    <>
      <Box>
        <Typography className={classes.text}>
          {text} {temp}&deg;
        </Typography>
      </Box>
    </>
  );
};

export default AsideTemp;
