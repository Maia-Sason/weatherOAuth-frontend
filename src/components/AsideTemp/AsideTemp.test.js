import React from "react";
import ReactDOM from "react-dom";
import { render } from "@testing-library/react";
import AsideTemp from "./AsideTemp";
import { ExpansionPanelActions } from "@material-ui/core";

test("<AsideTemp /> matches snapshot", () => {
  const component = render(<AsideTemp text={"Some text"} temp={67} />);
  expect(component.container).toMatchSnapshot();
});

it("Renders AsideTemp without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<AsideTemp />, div);
});
