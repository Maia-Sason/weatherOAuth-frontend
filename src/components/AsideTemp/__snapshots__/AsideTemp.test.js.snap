// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AsideTemp /> matches snapshot 1`] = `
<div>
  <div
    class="MuiBox-root MuiBox-root-2"
  >
    <p
      class="MuiTypography-root makeStyles-text-1 MuiTypography-body1"
    >
      Some text
       
      67
      Â°
    </p>
  </div>
</div>
`;
